.env.example

```diff
@@ -62,4 +62,7 @@ SUBSEQUENT_ALERT_MESSAGE_THRESHOLD_TEST=1000
 
 # How large a window we use to calculate moving average of radar values
 RADAR_WINDOW_SIZE_SECONDS=15
-RADAR_WINDOW_SIZE_SECONDS_TEST=1
\ No newline at end of file
+RADAR_WINDOW_SIZE_SECONDS_TEST=1
+
+# Access Token for particle device
+PARTICLE_ACCESS_TOKEN=particle_access_token
```

Comment:
Could you please add a `PARTICLE_ACCESS_TOKEN_TEST` dummy value under this as well? That will remind people to fill something in if they want to use it in their tests.